tests_integration/testing_utils/assertions/test_file_assertions.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+from syrupy import SnapshotAssertion
+
+
+def assert_that_current_working_directory_contains_only_following_files(
+    *filenames: str,
+) -> None:
+    assert set(os.listdir()) == set(filenames)
+
+
+def assert_that_file_content_is_equal_to_snapshot(
```

Comment:
Should we care about files content?

Reply:
IMO we should to avoid changing it by mistake (for example by replacing strings globally).

Reply:
I think this check is not needed, I can't find case when this content can change

Reply:
We copy files with external lib 

```
def copytree(src, dst, symlinks=False, ignore=None, copy_function=copy2,
             ignore_dangling_symlinks=False, dirs_exist_ok=False):
    """Recursively copy a directory tree and return the destination directory.
```

I think is safe :)

Reply:
We copy files using external lib, but we keep files in our repo (we can change files content by mistake). Maybe integration tests are not the best option for such check but what is the cost of having such check here?